cmd/provisioner-localpv/app/helper_hostpath.go

```diff
@@ -213,7 +213,28 @@ func (p *Provisioner) createCleanupPod(ctx context.Context, pOpts *HelperPodOpti
 
 	config.taints = pOpts.selectedNodeTaints
 
-	config.pOpts.cmdsForPath = append(config.pOpts.cmdsForPath, filepath.Join("/data/", config.volumeDir))
+	// check if path is xfs quota enabled and remove quota projid
```

Comment:
this is the fix for the non-distributed cleanup right? Could you place this on a separate commit?